src/actions/serve.js

```diff
@@ -1,14 +1,38 @@
 const execa = require('execa')
 const path = require('path')
+const fs = require('fs')
+const sequence = require('../helpers/sequence')
+const ask = require('../helpers/ask')
+
+const askFolder = list => ask('Which folder do you want to serve?', null, 'list', list)
 
 module.exports = () => {
-  const subprocess = execa(path.join(__dirname, '../..', 'node_modules/.bin/http-server'), [
-    './build',
-    '-o',
-    '-c-1',
-  ])
+  let folder
+  return sequence([
+    () => {
+      return ['./build', './dist/es5', './dist/es6', './dist/lightning++'].filter(f =>
```

Comment:
I don't think we should use the serve command for serving fro the distributable folder.